src/Graph/GraphFormatter.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace SRF\Graph;
+
+use Html;
+
+/**
+ *
+ *
+ * @see https://www.semantic-mediawiki.org/wiki/Help:Graph_format
+ *
+ * @license GNU GPL v2+
+ * @since 3.1
```

Comment:
Either 3.2 or just drop it

Reply:
@see: https://github.com/RacoonDev/SemanticResultFormats/commit/10eb56e4c1a888dd4f9988427e55bf3f12f98917